certified-connectors/PowerAppsCards/apiProperties.json

```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oauthSettings": {
+          "identityProvider": "aadcertificate",
+          "properties": {
+            "IsFirstParty": "True"
+          },
+          "scopes": [],
+          "clientId": "<<Enter client id>>",
+          "redirectMode": "Global",
+          "customParameters": {
+            "ResourceUri": {
+              "value": "https://management.core.windows.net/"
+            },
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            }
+          }
+        }
+      },
+      "token:TenantId": {
+        "type": "string",
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "uiDefinition": {
+          "displayName": "Login your Credentials",
+          "description": "Login your Credentials",
+          "tooltip": "Provide your Credentials",
+          "constraints": {
+            "required": "true",
+            "hidden": "false"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@GarimaSarangal this one of our default colors and it is not allowed to use. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector).

Reply:
updated the color